3cn-ecn/nantralPlatform
# Backend of favorite button

# Description

Added backend for favorite button

# Tests

1. Steps to reproduce :

- Run _pipenv run migrate_ to update the changes
- Make sure you are not connected as an admin account
- Make sure you have added event to the database 

2. Expected results :
- On press of the favorite button, the event will be added to your favorite. When refreshing the page, the favorite icon should stay selected.

# Screenshots
![image](https://user-images.githubusercontent.com/112650656/217004648-ea0159dd-fd3a-41aa-bbde-f41f5861fe3c.png)
![image](https://user-images.githubusercontent.com/112650656/217004722-26aa7324-2af8-4937-a19c-79464c5f7203.png)
